CHANGELOG.md

```diff
@@ -2,11 +2,12 @@
 
 ## (IN PROGRESS)
 
-## [2.0.0](https://github.com/folio-org/ui-gobi-settings/tree/v2.0.0) (2023-02-22)
+## [3.0.0](https://github.com/folio-org/ui-gobi-settings/tree/v2.0.0) (2023-02-22)
```

Comment:
2.0.0 is released, why do you change it?

Reply:
Can you check it please, if it is correct?